MAINTAINERS

```diff
@@ -13137,6 +13137,12 @@ S:	Maintained
 F:	Documentation/devicetree/bindings/phy/mediatek,mt7621-pci-phy.yaml
 F:	drivers/phy/ralink/phy-mt7621-pci.c
 
+MEDIATEK MIPS and Ralink CLOCK DRIVER
```

Comment:
Let's call this MEDIATEK MTMIPS CLOCK DRIVER.

Let's use MTMIPS as a family name for MediaTek MIPS and Ralink SoCs.

Do you intend to incorporate the mt7621 clock driver into this driver too?

Reply:
Mt7621 is a bit different allowing gating and other features that previous SoCs don't have. So I explained that also in clock driver commit message and was pretending to maintain that separate as it is now.